templates/security/cve/cve.html

```diff
@@ -213,19 +213,31 @@ <h2>Notes</h2>
         {% endif %}
 
         <h2>References</h2>
-        {% if cve.references %}
-        <ul>
-          {% for reference in cve.references %}
-          <li><a href="{{ reference }}">{{ reference }}</a></li>
-          {% endfor %}
-        </ul>
-        {% else %}
         <ul>
+          {% if cve.references %}
+            {% for reference in cve.references %}
+            <li><a href="{{ reference }}">{{ reference }}</a></li>
+            {% endfor %}
+          {% else %}
+            <li>
+              <a href="https://cve.mitre.org/cgi-bin/cvename.cgi?name={{ cve.id }}">
+                MITRE
+              </a>
+            </li>
+          {% endif %}
           <li>
-            <a href="https://cve.mitre.org/cgi-bin/cvename.cgi?name={{ cve.id }}">
-              MITRE
+            <a href="https://nvd.nist.gov/vuln/detail/{{ cve.id }}">
+              NVD
             </a>
           </li>
+          <li>
+            <a href="https://launchpad.net/bugs/cve/{{ cve.id }}">
+              Launchpad
+            </a>
+          </li>
+          <li>
+            <a href="https://security-tracker.debian.org/tracker/{{ cve.id }}">Debian</a>
+          </li>
         </ul>
         {% endif %}
```

Comment:
This breaks the layout, if I remove it the page loads, but not sure what this was closing